pages/dashboard.js

```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
   width: 100%;
 `;
 
-const getProfile = (data) => {
-  return data ? data.accountByEmail.data : [];
-}
-
 export default function Dashboard() {
   const user = useUser({ redirectTo: '/signup', redirectIfFound: false });
-  const { data, errorMessage } = getProfileByEmail('khobra.z@gmail.com');
-  const [ profile, setProfile ] = useState([]);
+  const [profile, setProfile] = useState(null);
+  const [email, setEmail] = useState(null);
+  const [applications, setApplications] = useState([]);
+  const [errorMessage, setErrorMessage] = useState(null);
   const { state } = useContext(Context);
 
-  const name = 'Zainab Ebrahimi';
-  const email = 'khobra.z@gmail.com';
-  const company = 'Tall Poppy';
-  const interviewDate = 'June 11 2021 @ 12pm ET';
-  const interviewType = 'Technical Interview';
-  const role = 'Software Engineer';
-  const excitement = '2';
-  const nerves = '2';
-
   useEffect(() => {
-    if (!profile.length) {
-      setProfile(getProfile(data))
+    async function fetchAccount() {
+      if (user?.email && !profile) {
+        const { data, errorMessage } = await accountByEmail(user.email);
+        const account = data ? data.accountByEmail.data[0] : null;
+        if (account) {
+          setProfile(account.profile);
+          setEmail(account.email);
+          setApplications(account.profile.applications.data);
+        }
+        if (errorMessage) {
+          setErrorMessage(errorMessage);
+        }
+      }
     }
-  }, [data, profile.length]);
+    fetchAccount();
+  }, [user, profile]);
+
+  return (user === undefined) ?
+    (
+      <>
```

Comment:
we dont need these fragments here given you are returning one component (in this case `<section>`)